README.md

```diff
@@ -51,6 +51,48 @@ GENESYSCLOUD_REGION
 
 For any issues, questions, or suggestions for the provider, visit the [Genesys Cloud Developer Forum](https://developer.mypurecloud.com/forum/)
 
+### Proxy Configuration
+
+Using of a proxy is accomplished by setting the proxy settings for the provider
+
+The `Proxy` has 3 properties that determine the URL for proxying.
+port - Port of the Proxy server
+host - Host Ip or DNS of the proxy server
+protocol - Protocol required to connect to the Proxy (http or https)
+
+The 'proxy' has another section which is an optional section. 
+If the proxy requires authentication to connect to
+'auth' needs to be mentioned under the 'Proxy'.
+
+The Provider Configuration with Proxy Looks like the below.
```

Comment:
Maybe something like "An example of the provider configuration with the proxy:" ? 

I would also put this in a code block like this:

\```hcl
provider "genesyscloud" {
    // ...
}
\```